src/pages/Registration/Registration.module.css

```
@@ -12,6 +12,60 @@
   align-items: center;
 }
 
+.previewPage {
+  background-color: #127BBE;
+  font-family: 'Merriweather', serif;
```

Stylelint (17.7.0)

[error] Expected no quotes around "Merriweather" (font-family-name-quotes)

(font-family-name-quotes)